src/js/structs/ServiceTree.js

```diff
@@ -168,21 +169,40 @@ module.exports = class ServiceTree extends Tree {
   }
 
   getServiceStatus() {
-    let {tasksRunning} = this.getTasksSummary();
-    let deployments = this.getDeployments();
+    // Return the service status with the highest index
+    let treeStatus = this.getStatusSummary().reduce((memo, count, statusKey) => {
```

Comment:
I realise the code is not very terse. I will think of a more self-explanatory way of doing the same thing.